### ISSUE TITLE
Additional archiveteam URL param

### ISSUE THREAD

**Issue by plexish** [WikiApiary/WikiApiary#172]
There are several websites I've turned up which have been archived at several different places. Do we want an additional archiveteam URL param? It would avoid losing information on connected wikis and help archiveteam deduplicate.

**Comment by jthingelstad** [comment]
Are you seeing additional archive URL's that are on Internet Archive? Or are they archives on other services?

**Comment by plexish** [comment]
[This wiki](https://wikiapiary.com/wiki/Marine_Biodiversity) and it's five duplicates all have different archive team URLs.

**Comment by plexish** [comment]
[Another](https://wikiapiary.com/wiki/ReachPlus_Alerts_Support_Wiki) [example](https://wikiapiary.com/wiki/ReachPlus_Alerts). Same wiki available from two different URLs archived by wikiteam in two different locations.

I'd like to deduplicate and use additional API URL rather than two pages, but would prefer not to disconnect from an IA archive which may be useful.

**Comment by nemobis** [comment]
etesp, 21/07/2014 15:59:

> Another https://wikiapiary.com/wiki/ReachPlus_Alerts_Support_Wiki
> example https://wikiapiary.com/wiki/ReachPlus_Alerts. Same wiki
> available from two different URLs archived by wikiteam in two different
> locations.

This for instance is a wiki that was migrated from one domain to the
other. (But is still accessible at two separate domains, one with and
one without www.)

**Comment by plexish** [comment]
@nemobis Indeed, and there's quite a few wikis in similar situations. What ways of avoiding duplication of archives and handling this kind of duplicate does wikiteam have currently? I was thinking WikiApiary could help out here, by collecting info queryable to generate a list of wikis you already have on another URL, to focus on wikis not recently archived more efficiently and maybe even merge existing archive records if there's a process for that.

**Comment by nemobis** [comment]
etesp, 21/07/2014 17:42:

> What ways of avoiding duplication of archives and handling this kind of
> duplicate does wikiteam have currently? 

Simple: none. I only remove the most obvious duplicates when I spot
them, and the biggest duplicate dumps when they get in my way.
SketchCow> [...] uploading stuff of dubious value or duplication to
archive.org: [...] gigabytes fine, tens of gigabytes problematic,
hundreds of gigabytes bad.

> I was thinking WikiApiary could
> help out here, by collecting info queryable to generate a list of wikis
> you already have on another URL, to focus on wikis not recently archived
> more efficiently 

Yes, that's definitely what WikiApiary is for I think. :)

> and maybe even merge existing archive records if
> there's a process for that.

Items on IA don't get merged, AFAICS.

**Comment by nemobis** [comment]
This is probably blocking https://github.com/WikiApiary/WikiApiary/issues/130 which in turn makes https://github.com/WikiTeam/wikiteam/issues/116 quite hard
